src/rgen/Microsoft.Macios.Generator.Sample/Microsoft.Macios.Generator.Sample.csproj

```diff
@@ -4,16 +4,23 @@
         <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
         <Nullable>enable</Nullable>
         <RootNamespace>Microsoft.Macios.Generator.Sample</RootNamespace>
+        <NoWarn>APL0003</NoWarn>
     </PropertyGroup>
 
     <ItemGroup>
         <ProjectReference Include="..\Microsoft.Macios.Generator\Microsoft.Macios.Generator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false"/>
     </ItemGroup>
 
     <ItemGroup>
+        <Compile Include="..\..\..\src\ObjCBindings\BindingTypeAttribute.cs" >
+            <Link>external\BindginTypeAttribute.cs</Link>
```

Comment:
Is `Bindgin` correct or should it be `Binding`?

Reply:
Typo!